### DISCUSSION TITLE
Offline status

### DISCUSSION THREAD

**Discussion by casper-none** [3urobeat/steam-comment-service-bot#218] [Q&A] [answered]
Can I make my bots offline status?

**Comment by 3urobeat** [comment]
Hey, sorry for the late response.  
At the moment no - your accounts will always appear online and you can only customize their playing status.  
This was a setting in the very first versions of this project but was removed at some point when I attempted to reduce the complexity of the config file - I should maybe reimplement it again (now that the advancedconfig exists anyway).  
I'll include this setting again in the next version!

**Reply by casper-none** [reply]
Ok, I will wait new version.

**Reply by 3urobeat** [reply, marked as answer]
I forgot to update this discussion, this setting has been reintroduced in [Version 2.15.0](https://github.com/3urobeat/steam-comment-service-bot/releases/tag/2.15.0) a few months ago.
